internal/export/interfaces/types.go

```diff
@@ -1,18 +1,15 @@
 //
-// Copyright (c) 2017
+// Copyright (c) 2018
 // Cavium
 // Mainflux
 // IOTech
 //
 // SPDX-License-Identifier: Apache-2.0
 //
 
-package distro
+package interfaces
```

Comment:
Why moving the interfaces to a new package? This interfaces are internal to export distro. No one else package are using them.
I should keep them in distro. And also should make the interfaces lowercase.

Reply:
It is because encryption.go in internal/export should be moved to internal/export/distro. Once it is moved the interfaces are not used anymore outside of distro.